reflex/constants.py

```diff
@@ -18,6 +19,28 @@
 IS_WINDOWS = platform.system() == "Windows"
 
 
+def get_fnm_name() -> Optional[str]:
+    """Get the appropriate fnm executable name based on the current platform.
+
+    Returns:
+        str: The fnm executable name for the current platform.
+    """
+    platform_os = platform.system()
+
+    if platform_os == "Windows":
+        return "fnm-windows"
+    elif platform_os == "Darwin":
+        return "fnm-macos"
```

Comment:
Why is there no ARM vs Intel distinction for Windows and Mac?

Reply:
i would assume macos is using fat binaries and windows is probably falling back to x86 emulation... but yeah, would definitely be good to test. i have windows arm, but i don't have x86 mac to test on.